account_move_group_restriction/models/account_account.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2025 Quartile (https://www.quartile.co)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+
+from odoo import fields, models
+
+
+class AccountAccount(models.Model):
+    _inherit = "account.account"
+
+    group_ids = fields.Many2many(
```

Comment:
```suggestion
    security_group_ids = fields.Many2many(
```
Is "group_id" a bit too generic a name?